draco/analysis/transform.py

```diff
@@ -453,8 +464,13 @@ def process(self, sstream):
         # variance for the m-modes
         weight_sum = sstream.weight[:].sum(axis=-1)
 
+        if self.set_telmmax is True:
```

Comment:
With that I said above this clause would just change to `self.telescope is not None`